compiler/dyno/include/chpl/parsing/parsing-queries.h

```diff
@@ -147,6 +148,18 @@ uast::Function::ReturnIntent idToFnReturnIntent(Context* context, ID id);
  */
 bool functionWithIdHasWhere(Context* context, ID id);
 
+void setConfigParams(Context* context, ConfigParamList keys);
+
+const
+ConfigParamList& configParams(Context* context);
+
+// TODO: Should these queries be private
+// use the config param by storing the ID where it was used
```

Comment:
I think we can have a better comment here -- something like `note the ID to which we have associated a config set on the compile command line`